### PR TITLE
Redesigned Invoices list item

### LINKED ISSUE #136: Redesign: /Payments/Invoices
**NOTICE** 

**Remember to checkout and send a pull request to the [redesign](https://github.com/openMF/mobile-wallet/tree/redesign)  branch.**

---

We are working on the app redesign. The prototype is done and can be found here: [show prototype](https://xd.adobe.com/view/8581e34d-8b91-49ec-7c21-000d98c8f18b-f3b0/?hints=off), and source files here: [show repository](https://github.com/letelete/mifos-pay-app-design). We will be working on the redesign implementation now.

This issue is about implementing the design and features of INVOICES tab in the Payments section.

That's what we want to achieve:

image

### PR DESCRIPTION
**Pull request by codepoet2017390**
Fixes #136 

**Screenshot for a list item**

screenshot_20190113-000152

Please Add Summary of what changes you have made.(Optional)
Redesigned the list item layout for invoices .

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by codepoet2017390** [comment]
@luckyman20  please review

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice/presenter/InvoicesPresenter.java [hunk 1]
```diff
@@ -44,14 +44,14 @@ public void fetchInvoices() {
...
-                        DebugUtil.log("ivoices fetched successfully",
+                        DebugUtil.log("invoices fetched successfully",
                                 response.getInvoiceList().size());
                         mInvoicesView.showInvoices(response.getInvoiceList());
                     }
 
                     @Override
                     public void onError(String message) {
-                        DebugUtil.log("unable to fetvh invoices");
+                        DebugUtil.log("unable to fetch invoices");
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice/presenter/InvoicesPresenter.java]: Do we need these log statements?

Comment (codepoet2017390) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice/presenter/InvoicesPresenter.java]: Yup i think its necessary.They were already there i just fixed the typo.

Comment (luckyman20) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice/presenter/InvoicesPresenter.java]: Log statements are only necessary for the developer while testing/debugging the app. Please remove them. I think we were merged by mistake.

Comment (codepoet2017390) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice/presenter/InvoicesPresenter.java]: Done

**Comment by ankurs287** [comment]
@codepoet2017390 good job

**Pull request merged**